Fix issues from maker.makecode.com for RP2040 (pico)

Simulator ServiceWorker registration successful with scope:  https://trg-maker.userpxt.io/
pxtworker.js:23 errors in inline assembly
pxtworker.js:23 At inline assembly:
  -> Line 19 ('    bl pins::digitalWritePin'), error: assembly error
   Maybe: bl    LABEL (unknown label at 'pins::digitalWritePin')
   Maybe: bls   LABEL (opcode name doesn't match at '<name>')
At inline assembly:
  -> Line 23 ('    bl pins::getPinAddress'), error: assembly error
   Maybe: bl    LABEL (unknown label at 'pins::getPinAddress')
   Maybe: bls   LABEL (opcode name doesn't match at '<name>')

pxtapp.js:1 error: pxt_modules/ws2812b/main.ts(1,1): error TS9200:   -> Line 19 ('    bl pins::digitalWritePin'), error: assembly error
   Maybe: bl    LABEL (unknown label at 'pins::digitalWritePin')
   Maybe: bls   LABEL (opcode name doesn't match at '<name>')


pxtapp.js:1 webusb: get devices
pxtworker.js:23 errors in inline assembly
pxtworker.js:23 At inline assembly:
  -> Line 19 ('    bl pins::digitalWritePin'), error: assembly error
   Maybe: bl    LABEL (unknown label at 'pins::digitalWritePin')
   Maybe: bls   LABEL (opcode name doesn't match at '<name>')
At inline assembly:
  -> Line 23 ('    bl pins::getPinAddress'), error: assembly error
   Maybe: bl    LABEL (unknown label at 'pins::getPinAddress')
   Maybe: bls   LABEL (opcode name doesn't match at '<name>')

pxtapp.js:1 error: pxt_modules/ws2812b/main.ts(1,1): error TS9200:   -> Line 19 ('    bl pins::digitalWritePin'), error: assembly error
   Maybe: bl    LABEL (unknown label at 'pins::digitalWritePin')
   Maybe: bls   LABEL (opcode name doesn't match at '<name>')

--- a/main.ts
+++ b/main.ts
@@ -25,6 +25,7 @@ namespace ws2812b {
     export function sendBuffer(buf: Buffer, pin: number) {
     }
 
+    //% shim=light::setMode
     /**
      * Sets the buffer mode
      */

--- a/pxt.json
+++ b/pxt.json
@@ -4,8 +4,7 @@
     "description": "A driver for WS2812B programmable LEDs in MakeCode",
     "license": "MIT",
     "dependencies": {
-        "core": "*",
-        "microphone": "*"
+        "core": "*"
     },
     "files": [
         "README.md",
@@ -28,5 +27,6 @@
                 }
             }
         }
-    }
+    },
+    "binaryonly": true
 }

